beacon-chain/blockchain/process_block.go

```diff
@@ -111,6 +112,30 @@ func (s *Service) onBlock(ctx context.Context, signed *ethpb.SignedBeaconBlock,
 		return err
 	}
 
+	// If slasher is configured, forward the attestations in the block via
+	// an event feed for processing.
+	if featureconfig.Get().EnableSlasher {
+		// Feed the indexed attestation to slasher if enabled. This action
+		// is done in the background to avoid adding more load to this critical code path.
+		go func() {
+			for _, att := range signed.Block.Body.Attestations {
+				committee, err := helpers.BeaconCommitteeFromState(preState, att.Data.Slot, att.Data.CommitteeIndex)
+				if err != nil {
+					log.WithError(err).Error("Could not get attestation committee")
+					traceutil.AnnotateError(span, err)
+					return
+				}
+				indexedAtt, err := attestationutil.ConvertToIndexed(ctx, att, committee)
```

Comment:
you should probably use another context given that its an async method which might be expensive.